server/protocols/jmap/src/main/java/org/apache/james/jmap/model/SetVacationRequest.java

```diff
@@ -45,7 +46,7 @@ public static Builder builder() {
 
         public Builder accountId(String accountId) {
             if (accountId != null) {
-                throw new NotImplementedException();
+                throw new JmapFieldNotSupportedException(COMMAND, "accountId");
```

Comment:
So... You are assuming you are being called by a method from inside a model?

That is a tendencious assertion, that breaks as soon as you start using the same model in several methods...

You should only say `The field 'accountId' of 'SetVacationRequest' is not supported`.